File: brand.alert/php/brand_alert_api_example.php
<?php

    $terms = 'cinema';
    $password = '*****';
    $username = 'username';
    $rows = '5';

    $url = "https://www.whoisxmlapi.com/brand-alert-api/search.php?"
        . "term1={$terms}&username={$username}&password={$password}&rows={$rows}";

    print(file_get_contents($url));

?>

File: brand.alert/ruby/brand_alert_api_example.rb
require 'open-uri'
require 'json'
require 'rexml/document'
require 'rexml/xpath'
require 'yaml'		# only needed to print the returned result in a very pretty way

########################
# Fill in your details #
########################
username = "YOUR_USERNAME"
password = "YOUR_PASSWORD"

term1 = "cinema"
exclude_term1 = 'movie'
exclude_term2 = 'online'

#######################
# Use a JSON resource #
#######################
format = "JSON"
url = 'https://www.whoisxmlapi.com/brand-alert-api/search.php?' +
    'term1=' + term1 +
    '&username=' + username +
    '&password=' + password +
    '&output_format=' + format +
    '&exclude_term1=' + exclude_term1
    '&exclude_term2=' + exclude_term2

# Open the resource
buffer = open(url).read

# Parse the JSON result
result = JSON.parse(buffer)

# Print out a nice informative string
puts "JSON:\n" + result.to_yaml + "\n"